src/RefractAPI.cc

```diff
@@ -153,7 +153,7 @@ namespace drafter {
 
         // Add default value
         if (!parameter.node->defaultValue.empty()) {
-            element->attributes[SerializeKey::Default] = CreateArrayElement(LiteralToRefract<T>(MAKE_NODE_INFO(parameter, defaultValue), context), true);
+            element->attributes[SerializeKey::Default] = LiteralToRefract<T>(MAKE_NODE_INFO(parameter, defaultValue), context);
```

Comment:
Okay I understand.

IMO it would have been clearer if API Elements didn't say that enum's default should be an array. There can only ever be one default and it doesn't make sense that it should be wrapped in an array.